.storybook/stories/config-pages/LocalFooter.stories.tsx

```diff
@@ -3,7 +3,9 @@ import type {Meta, StoryObj} from '@storybook/react';
 import LocalFooter from "@components/config-pages/local-footer";
 import {ComponentProps} from "react";
 
-type ComponentStoryProps = ComponentProps<typeof LocalFooter> & {}
+type ComponentStoryProps = ComponentProps<typeof LocalFooter> & {
+  numberoflinks: number
```

Comment:
For ease of reading and consistent code styles, it's good to use camel case characters. `numberOfLinks`

Reply:
I changed the name of `numberOfLinks` and noticed I had the same style problem in the gallery page so I fixed that too.